src/Catalyst.Core.Modules.Ledger.Tests/UnitTests/LedgerTests.cs

```diff
@@ -54,6 +58,11 @@ public sealed class LedgerTests : IDisposable
         private readonly ILedgerSynchroniser _ledgerSynchroniser;
         private readonly IHashProvider _hashProvider;
         private readonly MultiHash _genesisHash;
+        private readonly IKvm _kvm;
+        private readonly IDeltaExecutor _executor;
+        private readonly IStateProvider _stateProvider;
+        private readonly IStorageProvider _storageProvider;
+        private readonly ISpecProvider _specProvider;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_specProvider' or refactor the code to use its value.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4350015)